### ISSUE TITLE
Datepicker event - not accessible

### ISSUE THREAD

**Issue by tonifisler** [epfl-idevelop/elements#133]
The "Datepicker Event" (`/#/molecules/datepicker-event`) component is not accessible with a keyboard.

The plugin uses class `.picker__day--highlighted` to show the current item when navigating with arrow keys but it is removed in our code:

https://github.com/epfl-idevelop/elements/blob/e8dd85e45682596ade72dec27a8d936d83792480/assets/components/molecules/datepicker-event/datepicker-event.js#L164-L169

See the "Datepicker" (`/#/molecules/datepicker`) component for the correct and expected behaviour.
